windows/native_exec/simple_x64.py

```diff
@@ -1011,6 +1014,14 @@ class Mov(Instruction):
                 (RawBits.from_int(5, 0xb8 >> 3), X64RegisterSelector(), Imm64())]
 
 
+class Pushfd(Instruction):
+    encoding = [(RawBits.from_int(8, 0x9c),)]
+
+
+class Popfd(Instruction):
```

Comment:
Looks like the x64 instruction should be Popfq.
The DWORD equivalent cannot be encoded in 64 bits mode.